### PR TITLE
Stats rest implementation

### PR DESCRIPTION
**Pull request by ridicoulous**
#11 implementation. Added stats dto's, urls and tests

### CODE REVIEW

File: Bitmex.NET.IntegrationTests/Tests/BitmexApiServiceFundingApiTests.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace Bitmex.NET.IntegrationTests.Tests
+{
+    [TestClass]
+    [TestCategory("REST")]
+    public class BitmexApiServiceFundingApiTests : IntegrationTestsClass<IBitmexApiService>
+    {
+
+        [TestMethod]
+        public void should_return_fundings_list()
```
Comment (semashkinvg) [inline Bitmex.NET.IntegrationTests/Tests/BitmexApiServiceFundingApiTests.cs]: That might happen due to the settings in the account.  Can you please remove the test method and the class?

Comment (ridicoulous) [reply Bitmex.NET.IntegrationTests/Tests/BitmexApiServiceFundingApiTests.cs]: sure

**Comment by ridicoulous** [comment]
@semashkinvg take a look at QueryStringParams.cs. I included datetime fromatting, which those query is valid for server. Is it a good idea?

**Comment by semashkinvg** [comment]
@ridicoulous will do

**Comment by ridicoulous** [comment]
@semashkinvg What went wrong with orders testing? I'm adding comments from bitmex docs to classes and fields now

**Comment by ridicoulous** [comment]
Chat, Settelment, Schema will be later

**Pull request merged**